ejercicio1/form.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta name="author" content="bautistq">
+	<meta name="description" content="Sign up form">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Sign up form</title>
+</head>
+<body>
+	<section>
+		<h1>Form</h1>
+			<form>				
+				<label for="name">First Name</label>
```

Comment:
This `for` will not match with any control anyway might be a missed later since in the `lname` one is fine :)